packages/framework/src/Migrations/Version20240206145944.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20240206145944 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('DELETE FROM setting_values WHERE name = \'defaultAvailabilityInStockId\'');
```

Comment:
This whole migration file could be mentioned in the upgrade notes as it may be quite dangerous, what do you think?